packages/web3-eth-iban/types/tests/iban-tests.ts

```diff
@@ -0,0 +1,139 @@
+/*
+    This file is part of web3.js.
+    web3.js is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file iban-tests.ts
```

Comment:
I'm using pascal case in Web3.js. Check this also on the Utils PR.

Reply:
issue with this is that setting `tslint` to `"pascal-case"` `dslint` then wants `index.d.ts` to be `pascal-case` as well which does not work, as windows is not case sensitive but Linux is and this will cause huge issues on different systems (i.e. Index.D.ts on windows will work on Linux will not). This is why for testings and types i have used `kebab-case` + is what `ts` promote for typing file naming, if you look at DT you see basically every single type package is `kebab-case` 👍 let me know your thoughts :)